src/main/java/az/edu/turing/business/FamilyService.java

```diff
@@ -101,7 +101,7 @@ public Set<Pet> getPets(int familyIndex) {
         Family family = getFamilyById(familyIndex);
         if (family != null) {
             Set<Pet> pets = family.getPets();
-            return pets != null ? new HashSet<>(pets) : new HashSet<>();
+            return new HashSet<>(pets);
```

Comment:
just return pets no need new HashSet<> pets already Set

Reply:
Would it be a problem if I return a  reference directly?